core/state_processor.go

```diff
@@ -103,6 +108,10 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 		}
 		receipts = append(receipts, receipt)
 		allLogs = append(allLogs, receipt.Logs...)
+
+		if tx.Type() != types.DepositTxType && isJovian {
+			daFootprint += tx.RollupCostData().EstimatedDASize().Uint64() * params.DAFootprintGasScalar
+		}
```

Comment:
```suggestion
		if isJovian && tx.Type() != types.DepositTxType {
			daFootprint += tx.RollupCostData().EstimatedDASize().Uint64() * params.DAFootprintGasScalar
		}
```

Nit: fork-checks before any interactions with involved objects.

Reply:
this diff is removed in the follow up PR #675